### ISSUE TITLE
The model uses the Mamba module instead of the VSS module.

### ISSUE THREAD

**Issue by DongdongMeng** [MrBlankness/LightM-UNet#31]
The code for LightMUNet seems to be incorrect. You are using the same Mamba module as Umamba, instead of the VSS module. Please check the `class MambaLayer(nn.Module)` and respond to this, thank you.

**Comment by zs1314** [comment]
Yes!I think you are wrong!Please check the and respond to this, thank you.
